Resources/Assets.xcassets/mostRecentIcon.imageset/Contents.json

```diff
@@ -1,15 +1,21 @@
 {
   "images" : [
     {
-      "filename" : "3 Icons_2 Navigation_3 Business_02 Accounts.svg",
-      "idiom" : "universal"
+      "filename" : "mostPopular.svg",
```

Comment:
For SVG files we don't need to provide all 3 scales. SVGs are vector files so they don't "loose" quality when resizing. We can the use single scale option.

Reply:
I guess you've said it before, so I downloaded only .svg